packages/common-stellar/src/project/models.ts

```diff
@@ -87,6 +88,7 @@ export class TransactionHandler implements SubqlTransactionHandler {
 }
 
 export class SorobanTransactionHandler implements SubqlSorobanTransactionHandler {
+  @forbidNonWhitelisted({account: ''})
```

Comment:
What are these for?

Reply:
https://github.com/subquery/subql-stellar/pull/18#discussion_r1343377248

it could help us throw error on unknown filters